test/multiSamlStrategy.js

```diff
@@ -154,6 +151,7 @@ describe('strategy#logout', function() {
     var superLogoutMock = this.superLogoutMock;
     var samlOptions = {
       issuer: 'http://foo.issuer',
+      authnRequestBinding: 'HTTP-POST',
```

Comment:
Where do we validate that this has any effect? Where do we have a validation the the default behavior will still work?